frontend/shared-components/data-table/data-table.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from "react"
+
+import { useTable } from "react-table"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faGreaterThan, faPlusCircle } from "@fortawesome/free-solid-svg-icons"
+
+import styles from "./data-table.module.css"
+
+type IncidentData = {
+  dates: string
+  incidentType: string
+  officersInvolved: string[]
+  subject: string
+  source: string
+}
+
+export function DataTable({ count = 220375 }) {
+  // Defines table rows
```

Comment:
This doesn't seem like a necessary comment, I'm also not sure what it's referring to (`count`?) 

I think this would be better served by just giving its referent a more informative variable name (maybe `numTableRows`?)